.github/scripts/deploy.sh

```diff
@@ -3,6 +3,11 @@
 version=$(npm pkg get version | sed 's/"//g')
 bucket=$1
 
+versionsFile="../versions.txt";
+echo $version >> $versionsFile
+
+allVersions=$(<$versionsFile)
```

Comment:
issue: These changes to *versions.txt* won't persist.

Instead, could we read the current list of versions from s3 (i.e., `s3://$bucket/content/versions`)? This way the deployment bucket maintains the historic list of versions.

Reply:
Refactored